merlin/study/step.py

```diff
@@ -135,17 +136,17 @@ def clone_changing_workspace_and_cmd(
 
     def get_task_queue(self):
         """ Retrieve the task queue for the Step."""
-        return self.get_task_queue_from_dict(self.mstep.step.__dict__)
+        return QUEUE_TAG + self.get_task_queue_from_dict(self.mstep.step.__dict__)
 
     @staticmethod
     def get_task_queue_from_dict(step_dict):
         """ given a maestro step dict, get the task queue"""
+        queue = str(QUEUE_TAG)
         with suppress(TypeError, KeyError):
-            queue = step_dict["run"]["task_queue"]
-            if queue is None or queue.lower() == "none":
-                queue = "merlin"
-            return queue
-        return "merlin"
+            queue += step_dict["run"]["task_queue"]
+            if queue is None or queue.lower() == f"{QUEUE_TAG}none":
```

Comment:
Here, queue will never be 'None'. Best to check step_dict["run"]["task_queue"] first to see if that's None